devops/jobs/MissingNewRelicAlerts.groovy

```diff
@@ -0,0 +1,63 @@
+package devops.jobs
+import static org.edx.jenkins.dsl.Constants.common_wrappers
+
+class  MissingNewRelicAlerts {
+    public static def job = { dslFactory, extraVars ->
+        assert extraVars.containsKey("DEPLOYMENTS") : "Please define DEPLOYMENTS. It should be list of strings."
+        assert !(extraVars.get("DEPLOYMENTS") instanceof String) : "Make sure DEPLOYMENTS is a list of string"
+
+        extraVars.get('DEPLOYMENTS').each { deployment ->
+
+                dslFactory.job(extraVars.get("FOLDER_NAME","Monitoring") + "/missing-new-relic-alerts-${deployment}") {
+                    parameters {
+                        stringParam('CONFIGURATION_REPO', 'https://github.com/edx/configuration.git')
+                        stringParam('CONFIGURATION_BRANCH', 'missing_newrelic_alerts')
+                    }
+
+                    wrappers common_wrappers
+
+                    wrappers {
+                        credentialsBinding {
+                            string("NEW_RELIC_API_KEY", "${deployment}-new-relic-api-key")
+                            file("AWS_CONFIG_FILE","tools-edx-jenkins-aws-credentials")
+                            def variable = "${deployment}-missing-newrelic-alerts"
+                            string("ROLE_ARN", variable)
+                        }
+                    }
+
+                    triggers {
+                        cron("0 0 * * 0")
```

Comment:
Every night midnight utc on Sunday is that intentional?


I would atleast change the first 0 to an H
eg:
'H 0 * * 0'

This will distribute it to an arbitrary minute within the hour based on a hash and is a good practice for scheduling jenkins jobs so they dont all stack up.